src/flow/hooks.js

```diff
@@ -12,3 +12,27 @@ export const useElements = ({ data, config: { layout }, width, height }) => {
 
   return elements
 }
+
+
+function getWindowDimensions() {
+  const { innerWidth: width, innerHeight: height } = window
```

Comment:
would this always be of the entire window, instead of an inner element?

Reply:
Do you mean the entire page screen? I am worried that it would cause a problem with the current node distribution algorithm as it only needs the dimension of the frame where nodes are rendered.

Reply:
yea `window.innerWidth/innerHeight` as far as I know, is the dimension of the entire viewable window (browser), not the dimension of the container that hosts the viz

Reply:
I believe it depends on where it is called. Currently, it returns the dimension of div the hook is called.

Reply:
will test it out